src/main/java/net/pms/formats/OGG.java

```diff
@@ -0,0 +1,27 @@
+package net.pms.formats;
+
+public class OGG extends Format {
+
+	@Override
+	public Identifier getIdentifier() {
+		return Identifier.OGG;
+	}
+
+	@Override
+	public boolean transcodable() {
+		return true;
+	}
+
+	public OGG() {
+		type = VIDEO;
+	}
+
+	@Override
+	public String[] getSupportedExtensions() {
+		return new String[] {
+			"ogg",
+			"ogm",
+			"ogv"
```

Comment:
I know, and it was not a critic, just a reminder to make sure that you're aware of it.
Me also, most of the time quick merge make me that is was not perfect.

That's said, i think that you're dreaming about AAC(M4A) support fix, as for me most of the users loose their ability to play it on their physical renderer anymore.